.gitignore

```diff
@@ -39,3 +39,9 @@ coverage.xml
 /docs/src/
 /docs/build/
 /docs/source/generated/
+
+# Jupyter
+examples/notebooks/.ipynb_checkpoints/
+
+# System files
+.DS_Store
```

Comment:
Let's keep this file as it was, please.
You can ignore `.ipynb_checkpoints` and `.DS_Store` locally with `.git/info/exclude` ... We haven't yet decided on if we should exclude these files repository-wise...